colorcoder/Main.java

```diff
@@ -1,120 +1,123 @@
-        return new ColorPair(majorColor, minorColor);
-    }
-    static int GetPairNumberFromColor(MajorColor major, MinorColor minor) {
-        return major.getIndex() * numberOfMinorColors + minor.getIndex() + 1;
-    }
+    // static ColorPair GetColorFromPairNumber(int pairNumber) {
+    //     int zeroBasedPairNumber = pairNumber - 1;
+    //     MajorColor majorColor = 
+    //         MajorColor.fromIndex(zeroBasedPairNumber / numberOfMinorColors);
+    //     MinorColor minorColor =
+    //         MinorColor.fromIndex(zeroBasedPairNumber % numberOfMinorColors);
+    //     return new ColorPair(majorColor, minorColor);
+    // }
+    // static int GetPairNumberFromColor(MajorColor major, MinorColor minor) {
+    //     return major.getIndex() * numberOfMinorColors + minor.getIndex() + 1;
+    // }
 
-    static void testNumberToPair(int pairNumber,
-        MajorColor expectedMajor,
-        MinorColor expectedMinor)
-    {
-        ColorPair colorPair = GetColorFromPairNumber(pairNumber);
-        System.out.println("Got pair " + colorPair.ToString());
-        assert(colorPair.getMajor() == expectedMajor);
-        assert(colorPair.getMinor() == expectedMinor);
-    }
+    // static void testNumberToPair(int pairNumber,
+    //     MajorColor expectedMajor,
+    //     MinorColor expectedMinor)
+    // {
+    //     ColorPair colorPair = GetColorFromPairNumber(pairNumber);
+    //     System.out.println("Got pair " + colorPair.ToString());
+    //     assert(colorPair.getMajor() == expectedMajor);
+    //     assert(colorPair.getMinor() == expectedMinor);
+    // }
 
-    static void testPairToNumber(
-        MajorColor major,
-        MinorColor minor,
-        int expectedPairNumber)
-    {
-        int pairNumber = GetPairNumberFromColor(major, minor);
-        System.out.println("Got pair number " + pairNumber);
-        assert(pairNumber == expectedPairNumber);
-    }
+    // static void testPairToNumber(
+    //     MajorColor major,
+    //     MinorColor minor,
+    //     int expectedPairNumber)
+    // {
+    //     int pairNumber = GetPairNumberFromColor(major, minor);
+    //     System.out.println("Got pair number " + pairNumber);
+    //     assert(pairNumber == expectedPairNumber);
+    // }
 
     public static void main(String[] args) {
-        testNumberToPair(4, MajorColor.WHITE, MinorColor.BROWN);
-        testNumberToPair(5, MajorColor.WHITE, MinorColor.SLATE);
+         ColorPairTest.testNumberToPair(4, MajorColor.WHITE, MinorColor.BROWN);
+         ColorPairTest.testNumberToPair(5, MajorColor.WHITE, MinorColor.SLATE);
     
-        testPairToNumber(MajorColor.BLACK, MinorColor.ORANGE, 12);
-        testPairToNumber(MajorColor.VIOLET, MinorColor.SLATE, 25);
+         ColorPairTest.testPairToNumber(MajorColor.BLACK, MinorColor.ORANGE, 12);
+         ColorPairTest.testPairToNumber(MajorColor.VIOLET, MinorColor.SLATE, 25);
+         System.out.println(ColorPairUtils.getColorCodeManual());
```

Comment:
The reference manual does not have a `test...` function. The absence of asserts means the entire output needs to be examined manually. Can you think of any asserts to reduce the burden on manual inspection?